src/components/App/Menu/AppMenu.vue

```diff
@@ -248,34 +248,13 @@ export default class AppMenu extends Mixins(TranslationMixin) {
       background-color: unset !important;
     }
   }
-}
-
-// TODO: [TECH] move from fonts provided values
-.sora-card-sidebar-icon {
-  path {
-    fill: var(--s-color-base-content-tertiary) !important;
-  }
-}
 
-.el-menu-item:not(.is-active):not(.is-disabled) {
-  .sidebar-item-content {
-    &:hover .sora-card-sidebar-icon path {
-      fill: var(--s-color-base-content-secondary) !important;
+  .el-menu-item {
+    i.el-icon-bank-card {
+      width: 28px;
```

Comment:
size=28 prop instead

Reply:
<img width="1085" alt="Screenshot 2023-10-30 at 19 50 18" src="https://github.com/sora-xor/polkaswap-exchange-web/assets/27702061/b126c7c3-35ac-411c-9999-8b9b7e54f81a">

it's already like so, it has no effect. The structure is a bit different when el-icons